Update missing entries in changelog (#5915)
rotki/rotki: docs/changelog.rst

* :feature:`5588` Users will now be able to save and restore used filters in the history section.
* :feature:`5587` Forward/Backward navigation for filters/pagination (using mouse buttons) will now be possible in the history section.
* :feature:`5569` It is now possible to detect when an evm address is used in other chains and start tracking it. Also a background task runs periodically that performs this check.

> * :feature:`3004` Users will now be able to automatically track balances in convex gauges and their CVX locked and staked.

Reviewer: ```suggestion
* :feature:`3004` Users will now be able to automatically track balances in convex gauges and their locked and staked CVX.
```

> * :feature:`5611` Users will now see an improved notification if their API keys for Etherscan are missing and will be easier to take action on it.

Reviewer: ```suggestion
* :feature:`5611` Users will now see an improved notification if their API keys for Etherscan are missing and it will be easier to take action on it.
```

* :bug:`-` Fixed an edge case where removing an EVM account multiple times in a row, while a transactions querying task ran, would result in an error.
* :bug:`-` Ignoring forked assets ETC, BCH and BSV for accounting should now also remove any pre-fork references of them and completely omit them from the PnL report.
* :bug:`-` Users with kraken accounts with old data that were never purged and repulled will no longer have missing events.

> * :bug:`-` Fixed a bug that didn't allow to add a new Bitpanda account.

Reviewer: ```suggestion
* :bug:`-` Fixed a bug that didn't allow adding a new Bitpanda account.
```
